portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/layout/pagetemplate/masterspagetemplate/MastersPageTemplates.testcase

```diff
@@ -1710,4 +1710,62 @@ definition {
 		}
 	}
 
+	@description = "This is for LPS-133450. View the master page reverts to default. Test reported in bug LPS-131982."
+	@priority = "3"
+	test ViewMastersPageReturnToDefault {
+		task ("Add a page based on custom master") {
+			MastersPageTemplatesAdmin.gotoMasters(masterLayoutName = "Custom Masters Page Template");
+
+			Button.click(button = "Publish Master");
+
+			SitePages.openPagesAdmin(siteURLKey = "test-site-name");
+
+			ContentPages.addPage(
+				masterLayoutName = "Custom Masters Page Template",
+				pageName = "Test Page Name");
+		}
+
+		task ("Assert in the Master tab if the Custom Masters Page Template is selected") {
+			PageEditor.gotoTab(tabName = "Page Design Options");
+
+			Click(locator1 = "PageEditor#MORE_LINK");
```

Comment:
Should be
`		AssertElementPresent(
			key_card = "${masterLayoutName}",
			locator1 = "Card#CARD_SELECTED");
`